homeassistant/components/google/calendar.py

```diff
@@ -3,6 +3,10 @@
 import logging
 
 from homeassistant.components.calendar import CalendarEventDevice
+from homeassistant.components.google import (
+    CONF_CAL_ID, CONF_ENTITIES, CONF_TRACK, TOKEN_FILE,
+    CONF_IGNORE_AVAILABILITY, CONF_SEARCH, CONF_MAX_RESULTS,
```

Comment:
Move the import of `CONF_MAX_RESULTS` below to the relative import group. Remove the rest of the imports here from the google package. They are imported below.

Reply:
I have no idea what you mean by relative import group but I also don't think CONF_MAX_RESULTS should be any different than the other CONF elements. Please elaborate...

Reply:
Check line 12 in the module. There we make a relative import from the google component package where we import all the things that are imported here on line 6, except `CONF_MAX_RESULTS`. So just move import of `CONF_MAX_RESULTS` to that the relative imports and remove the imports here.

Reply:
OK. Do I remove line 6 completely after fixing line 12? Forgive me. I mostly develop hass frontend...

Reply:
Yes.

Reply:
Done